bugsnag-android-core/src/test/java/com/bugsnag/android/ConfigurationTest.java

```diff
@@ -180,4 +181,44 @@ public void testVersionCode() {
         configuration.setVersionCode(577);
         assertEquals(577, (int) configuration.getVersionCode());
     }
+
+    @Test
+    public void testAddMetadata() {
```

Comment:
I guess just a head's up that some rationalisation will be needed whenever the changes meet the null check changes that Jamie currently has in review.

Reply:
I don't believe the additional null checking should invalidate any current behaviour, but we can hold this PR until the change has been merged and re-review if necessary.

Reply:
Sorry, didn't mean any behaviour would be invalidated, it's just that he's created a new ConfigurationFacadeTest.java file that we may want to merge with.
https://github.com/bugsnag/bugsnag-android/pull/803/files#diff-9bae5430ee1c302bbe1fcbe38404051d